webdriver-spec.html

```diff
@@ -5346,42 +5346,28 @@ <h3 id="character-types">Character types</h3>
 <section class=appendix>
 <h3>Security</h3>
 
-<p class=issue><strong>Warning:</strong>
- This section has not yet been redefined to match the <a>routing requests</a> model,
- and uses old concepts and definitions.
- Please do not rely on it yet.
-
-<p>The following section is non-normative.
-
-<p>The remote end SHOULD have a specific command line argument
- when starting the browser up if it is connecting to a browser
- (e.g. firefox.exe --webdriver)
- and SHOULD have user agent configuration preference
- that is tested when the user agent starts up.
-
-<p>When the remote end starts up, it MUST include a flag or setting
- to limit the IP addesses allowed to connect and create sessions.
- The default value for this setting SHOULD limit connections
- to be from <code>127.0.0.1</code> IPV4 address
- or <code>::1</code> IPV6 address.
- This will prevent arbitrary machines from connecting
- and creating <a data-lt=session>WebDriver sessions</a>
- unless specifically configured to allow them,
- while still supporting use-cases where the <a>local end</a>
- runs on a different machine from the <a>remote end</a>.
-
-<aside class=example>
- <p>Some devices will require the device
-  to be placed into a “developer mode”.
-  This allows Unix sockets to be forwarded on to the device
-  while the device is connected to a host.
-  An example of this is if the local end needed to go through
-  a remote debugger protocol.
-</aside>
-
-<p>If any of these fail then a
- <code><a href="#status-session-not-created">session not created</a></code>
- error MUST be thrown when the local end tries to create a new session.
+<p>If the <a>remote end</a> and the user agent
+ occupy the same binary executable,
```

Comment:
What I’m trying to say here is that if the WebDriver server lives inside the browser’s binary it shouldn’t ship with WebDriver enabled by default.  If the remote end binary is separate, no such restriction is needed.

I’m unsure how to make this clearer… suggestions?